pkg/compose/tests/fixtures/v3-simple/output.regexp-golden

```diff
@@ -0,0 +1,52 @@
+CMD: stack deploy -c pkg/compose/tests/fixtures/v3-simple/docker-compose.yml v3-simple
+Deployed Stack {{.*}}
+
+CMD: stack services v3-simple
+no services found in stack: v3-simple
```

Comment:
This is wrong, but the back-end isn't wired up yet.  As soon as it is, then this would fail and we can fill in the details with appropriate regex for the service IDs.